CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+### Fixed
+* output-plugins routes must be bound *before* provider routes
```

Comment:
needs a test

Reply:
Added in a separate PR (https://github.com/koopjs/koop-core/pull/76) since this was already merged.